### PR TITLE
[WIP] Update documentation to new data.camera()

### PR DESCRIPTION
**Pull request by kalpanab-psg**
## Description

Fixes: #4993 

## Checklist

- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- Gallery example in `./doc/examples` (new features only)
- Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- Unit tests
- Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

File: doc/examples/segmentation/plot_multiotsu.py [hunk 1]
```diff
@@ -53,7 +53,7 @@
     ax[1].axvline(thresh, color='r')
 
 # Plotting the Multi Otsu result.
-ax[2].imshow(regions, cmap='Accent')
+ax[2].imshow(regions, cmap='jet')
```
Comment (jni) [inline doc/examples/segmentation/plot_multiotsu.py:56]: I think this change is probably not necessary?

**Comment by jni** [comment]
@kalpanab-psg Thanks for these! I'm not sure why the circleci builds failed. I also don't understand why I need to log in to view the Circle-CI build log — @scikit-image/core any ideas?

But, since we can't see the output online, would you mind posting screenshots of the figure outputs of your changes here, to make review easier?

File: doc/examples/segmentation/plot_multiotsu.py (hunk 1 above)
Comment (kalpanab-psg) [reply doc/examples/segmentation/plot_multiotsu.py:56]: I think someone mentioned a color change to lessen the pink effect here ->https://github.com/scikit-image/scikit-image/issues/4993#issuecomment-705545483.
The cmap with jet parameter looks better than Accent.

**Comment by kalpanab-psg** [comment]
Multi otsu
multi_otsu

Floodfill - seed point changed to (150,155)
floodfill


watershed-- image changed to eagle
watershed

File: doc/examples/segmentation/plot_multiotsu.py (hunk 1 above)
Comment (jni) [reply doc/examples/segmentation/plot_multiotsu.py:56]: Ah, fair enough. I just have a severe allergy to jet. LOL I think even if not accent we should use one of the **qualitative** colormaps found in [this page](https://matplotlib.org/3.1.1/tutorials/colors/colormaps.html)

**Review by jni — APPROVED** [review]
Thanks so much for the screenshots! I've suggested a minor code fix re seed point, and an optional suggestion about the multiotsu colormap — not something I feel that strongly about.

File: doc/examples/segmentation/plot_floodfill.py [hunk 2]
```diff
@@ -54,16 +54,16 @@
...
+light_coat = flood_fill(cameraman, (150, 155), 255, tolerance=10)
 fig, ax = plt.subplots(ncols=2, figsize=(10, 5))
 
 ax[0].imshow(cameraman, cmap=plt.cm.gray)
 ax[0].set_title('Original')
 ax[0].axis('off')
 
 ax[1].imshow(light_coat, cmap=plt.cm.gray)
-ax[1].plot(150, 170, 'ro')  # seed point
+ax[1].plot(150, 155, 'ro')  # seed point
```
[the remark this answers is not in the document]
Comment (kalpanab-psg) [reply doc/examples/segmentation/plot_floodfill.py:66]: But when we see the output image the (x,y) values are showed as  (150,155) in red dot.

[the remark this answers is not in the document]
Comment (kalpanab-psg) [reply doc/examples/segmentation/plot_floodfill.py:66]: Will check and revert regarding other cmap option. Thanks for the suggestions

**Comment by jni** [comment]
@kalpanab-psg thanks for the fix! I'm going to merge this now! If you find a better colormap, please open a new PR. =)

**Pull request merged**